lxd/storage/drivers/driver_zfs.go

```diff
@@ -20,6 +20,7 @@ import (
 	"github.com/lxc/lxd/shared/version"
 )
 
+var zfsAllowedFilesystems = []string{"btrfs", "ext4", "xfs"}
```

Comment:
I wonder if we should unify this with `cephAllowedFilesystems` and have a single slice called `blockBackedAllowedFilesystems` rather than duplicating this?

Reply:
I agree. There's also `lvmAllowedFilesystems` with the same values.

Reply:
Cool, lets unify all 3.